CHANGELOG.md

```diff
@@ -22,7 +22,13 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-2.2.4...master
+[Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-2.2.6...master
+
+## [2.2.6] - 2021-09-08
+[2.2.6]: https://github.com/atlassian-labs/db-replica/compare/release-2.2.4...release-2.2.6
+
+### Fixed
+- Additional caching in `AuroraClusterDiscovery`
```

Comment:
Shouldn't be AuroraMultiReplicaConsistency? `AuroraClusterDiscovery` is an internal.